code/view/json.php

```diff
@@ -135,6 +135,12 @@ protected function _actionRender(ViewContext $context)
      */
     protected function _fetchData(ViewContext $context)
     {
+        $content = $this->getContent();
```

Comment:
```suggestion
        $content = $context->content;
```
@jebbdomingo can you try the above suggestion? It should play better with other behaviors that could affect the content

Reply:
@ercanozkaya `$context->content` is `null`

Reply:
@jebbdomingo Can you please show me the code in Docman that is affected by this so that I can properly check it out ?

Reply:
@ercanozkaya this one in submit controller https://github.com/joomlatools/foliolabs-docman/blob/465714568990d3ea546b66edbad31784aeb135f0/code/site/controller/submit.php#L84